app/views/shared/ubiquity/contact/_new.html.erb

```diff
@@ -11,6 +11,7 @@
 <% message_value = params[:message_value] %>
 <% if user_signed_in? %>
   <% nm = current_user.name %>
```

Comment:
Please confirm If we have to fetch the name from **current_account** instead of _current_user_

Reply:
done